### PR TITLE
Update rubygem-safemode to 1.3.4

### PR DESCRIPTION
**Pull request by witlessbird**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.17
* [x] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by witlessbird** [comment]
This requires new versions of sexp_processor, ruby2ruby, and ruby_parser gems

**Comment by ekohl** [comment]
Needs https://github.com/theforeman/foreman-packaging/pull/2103, https://github.com/theforeman/foreman-packaging/pull/2104 and https://github.com/theforeman/foreman-packaging/pull/2105

File: rubygem-safemode/rubygem-safemode.spec [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
 License: MIT
 URL: https://github.com/svenfuchs/safemode
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}rubygems
-Requires: %{?scl_prefix}rubygem(ruby2ruby) >= 2.0.6
-Requires: %{?scl_prefix}rubygem(ruby_parser) >= 3.2.0
-Requires: %{?scl_prefix}rubygem(sexp_processor) >= 4.3.0
+Requires: %{?scl_prefix}rubygem(ruby2ruby) >= 2.4.0
+Requires: %{?scl_prefix}rubygem(ruby_parser) >= 3.10.1
```
Comment (ekohl) [inline rubygem-safemode/rubygem-safemode.spec]: This and sexp_processor aren't updated in the gemspec itself. Is that an oversight? I'd prefer to keep this in sync with the gemspec if possible

Comment (witlessbird) [reply rubygem-safemode/rubygem-safemode.spec]: Yeah, I missed that, thanks for catching it.

**Comment by witlessbird** [comment]
* fixed dependency versions

**Comment by ekohl** [comment]
I see you want these picked into 1.16. Does that bring some bugfix or other feature? I'm somewhat hesitant to push this into a released version.

**Comment by ekohl** [comment]
PR to remove the pins: https://github.com/theforeman/foreman/pull/5190
and https://github.com/theforeman/foreman-packaging/pull/2113

**Pull request merged**